src/AspNet.Security.OAuth.SuperOffice/Implementation/DefaultSuperOfficeConfigurationManager.cs

```diff
@@ -0,0 +1,101 @@
+            {
+                _logger.LogInformation($"Loading SuperOffice JwksUri from {Configuration.JwksUri}.");
+
+                _publicKey = await LoadJwksAsync(context, Configuration.JwksUri);
+                _reloadKeysAfter = utcNow.Add(context.Options.PublicKeyCacheLifetime);
+
+                _logger.LogInformation(
+                    $"Loaded SuperOffice JwksUrl from {Configuration.JwksUri} and obtained public keys. Keys will be reloaded at or after {_reloadKeysAfter}.");
+            }
+
+            return JsonWebKeySet.Create(_publicKey);
+        }
+
+        /// <inheritdoc />
+        public override async Task LoadConfigurationAsync(SuperOfficeValidateIdTokenContext context)
+        {
+            using var response = await context.Options.Backchannel.GetAsync(context.Options.ConfigurationEndpoint, context.HttpContext.RequestAborted);
+
+            if (!response.IsSuccessStatusCode)
+            {
+                _logger.LogError("An error occurred while retrieving the configuration from SuperOffice: the remote server " +
+                                 "returned a {Status} response with the following payload: {Headers} {Body}.",
+                                 /* Status: */ response.StatusCode,
+                                 /* Headers: */ response.Headers.ToString(),
+                                 /* Body: */ await response.Content.ReadAsStringAsync());
+
+                throw new HttpRequestException("An error occurred while retrieving the configuration from SuperOffice.");
+            }
+
+            byte[] jsonBytes = await response.Content.ReadAsByteArrayAsync();
+            Configuration = GetConfiguration(jsonBytes);
+        }
+
+        private async Task<string> LoadJwksAsync(SuperOfficeValidateIdTokenContext context, string jwksUrl)
+        {
+            using var response = await context.Options.Backchannel.GetAsync(jwksUrl, context.HttpContext.RequestAborted);
+
+            if (!response.IsSuccessStatusCode)
+            {
+                _logger.LogError("An error occurred while retrieving the keys from SuperOffice: the remote server " +
+                                 "returned a {Status} response with the following payload: {Headers} {Body}.",
+                                 /* Status: */ response.StatusCode,
+                                 /* Headers: */ response.Headers.ToString(),
+                                 /* Body: */ await response.Content.ReadAsStringAsync());
+
+                throw new HttpRequestException("An error occurred while retrieving the keys from SuperOffice.");
+            }
+
+            return await response.Content.ReadAsStringAsync();
+        }
+
+        private static SuperOfficeAuthenticationConfiguration GetConfiguration(byte[] jsonAsBytes)
+        {
+            var jsonSpan = new ReadOnlySpan<byte>(jsonAsBytes);
+            return JsonSerializer.Deserialize<SuperOfficeAuthenticationConfiguration>(jsonSpan);
```

Comment:
`byte[]` is implicitly cast as a `Span<byte>`, so this is redundant,

```suggestion
            return JsonSerializer.Deserialize<SuperOfficeAuthenticationConfiguration>(jsonAsBytes);
```